independent-publisher-connectors/Monster API/apiDefinition.swagger.json

```diff
@@ -0,0 +1,765 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Monster API",

```

Comment:
Hello @troystaylor,

Our validator is not letting us queue this up. it is complaining about the API in the title. can you please try to change it? maybe something like "MonsterAPI"!

Reply:
@troystaylor apologies for this but actually it went through today with the old title, so you can revert this change back :-)